apps/agora/data/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM python:3.11.9-slim
+
+ARG LOCAL_DATA_DIR="local/data"
```

Comment:
```suggestion
ARG LOCAL_DATA_DIR="./local/data"
```

Consider prefixing with `./` to indicate more clearly that its a relative path.

Reply:
Done in the next commit